### ISSUE TITLE
Promote Applications in Any Namespace to Stable

### ISSUE THREAD

**Issue by gnunn1** [argoproj/argo-cd#16189]
# Summary

Promote Application in Any Namespace from Beta to Stable

# Motivation

The Applications in Any Namespace feature was introduced in 2.5 and since then considerable work has been done to correct outstanding issues. At the moment there are a handful of issues and none of them appear to be of the showstopping variety.

This feature has been widely adopted by the community as it addresses some significant limitations with operating Argo CD in a multi-tenant environment. This broad adoption belies it's Beta status and strengthens the argument that it should be promoted to a Stable feature.

# Proposal

Applications in Any Namespace should be promoted to Stable where it makes sense with the Argo release cycle, ideally as part of the current 2.9 release cycle but worse case 2.10.

**Comment by crenshaw-dev** [comment]
I'm in support of this. The feature seems to be behaving well.

**Comment by jannfis** [comment]
Agreed. I believe most of the early childhood flaws have been remedied as of now. I'd have a good feeling promoting it to stable.

**Comment by crenshaw-dev** [comment]
We should probably make sure appsets-in-any-namespace is still marked alpha or beta.

**Comment by jannfis** [comment]
One thing that I (and potentially others) have overlooked, there is no way yet to configure this via CLI or UI, I believe.

**Comment by gnunn1** [comment]
As an FYI, I think the CLI and UI are both there now?

Merged PR for UI: https://github.com/argoproj/argo-cd/pull/16249
Code for CLI (As part of Apps in Any Namespace original PR from @jannfis): https://github.com/argoproj/argo-cd/pull/9755

**Comment by agaudreault** [comment]
+1 If everything is ready here, we just need a kind soul to open a PR and update all the doc related to this feature.

**Comment by Mangaal** [comment]
Application in Any Namespace is now GA, closing this issue.
 https://argo-cd.readthedocs.io/en/stable/operator-manual/app-any-namespace/Applications
